<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Bharani" failures="1" tests="3" name="Test" time="19.573" errors="0" timestamp="2022-10-11T08:51:41 IST">
  <testcase classname="testng.Testcase1" name="login" time="0.242">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [we2343454rty] but found [actiTIME - Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [we2343454rty] but found [actiTIME - Login]
at testng.Testcase1.login(Testcase1.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="testng.Testcase2" name="forgotPassowrd" time="0.254"/>
  <testcase classname="testng.Testcase3" name="actiitmeinx" time="0.098"/>
</testsuite> <!-- Test -->
